pyvisa_py/tcpip.py

```diff
@@ -59,6 +59,9 @@ class TCPIPInstrSession(Session):
     # for a specific kind of resource
     parsed: rname.TCPIPInstr
 
+    # Setting if keepalive has been activated
+    keepalive: bool
+
```

Comment:
I'm not sure if this is the right to set this or if attributes are handled internally?

Reply:
It looks fine to me.